projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/case-flag/write-case-flag-field.component.spec.ts

```diff
@@ -1,15 +1,20 @@
+import { Location } from '@angular/common';
 import { CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
-import { waitForAsync, ComponentFixture, TestBed } from '@angular/core/testing';
+import { ComponentFixture, TestBed, waitForAsync } from '@angular/core/testing';
 import { FormControl, FormGroup, ReactiveFormsModule } from '@angular/forms';
 import { ActivatedRoute } from '@angular/router';
+import { CaseEditDataService } from '../../../commons/case-edit-data';
 import { CaseField, FieldType } from '../../../domain/definition';
 import { CaseFlagState, FlagDetailDisplayWithFormGroupPath, FlagsWithFormGroupPath } from './domain';
 import { CaseFlagFieldState, CaseFlagStatus } from './enums';
 import { WriteCaseFlagFieldComponent } from './write-case-flag-field.component';
 
 import createSpy = jasmine.createSpy;
+import * as _ from 'lodash';
```

Comment:
@MateiHMCTS - consider removing `lodash` import if it's not being used anywhere

Reply:
I use it one line 329
        { provide: ActivatedRoute, useValue: _.cloneDeep(mockRoute) },
Need to clone deep as test data is declared outside beforeEach; alternative could be to move everything there... I think it is fine to leave it like this for now?

Reply:
@MateiHMCTS - yeah, that's fine... thanks for checking